decidim-generators/spec/runtime/queue_flag_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+if ENV["SIMPLECOV"]
+  require "simplecov"
+
+  SimpleCov.add_filter "/lib/decidim/generators/app_templates/"
+  SimpleCov.add_filter "/lib/decidim/generators/component_templates/"
+end
+
+require "spec_helper"
+require "json"
+require "fileutils"
+require "decidim/gem_manager"
+require "decidim/generators/test/generator_examples"
+
+module Decidim
+  describe Generators do
+    let(:env) do |example|
+      #
+      # When tracking coverage, make sure the ruby environment points to the
+      # local version, so we get the benefits of running `decidim` directly
+      # without `bundler` (more realistic test), but also get code coverage
+      # properly measured (we track coverage on the local version and not on the
+      # installed version).
+      #
+      if ENV["SIMPLECOV"]
+        {
+          "RUBYOPT" => "-rsimplecov #{ENV.fetch("RUBYOPT", nil)}",
+          "RUBYLIB" => "#{repo_root}/decidim-generators/lib:#{ENV.fetch("RUBYLIB", nil)}",
+          "PATH" => "#{repo_root}/decidim-generators/exe:#{ENV.fetch("PATH", nil)}",
+          "COMMAND_NAME" => example.full_description.tr(" ", "_")
+        }
+      else
+        {}
+      end
+    end
+
+    let(:result) do
+      Bundler.with_original_env { GemManager.capture(command, env:) }
+    end
+
+    # rubocop:disable RSpec/BeforeAfterAll
+    before(:all) do
+      Bundler.with_original_env { Decidim::GemManager.install_all(out: File::NULL) }
+    end
+
+    after(:all) do
+      Bundler.with_original_env { Decidim::GemManager.uninstall_all(out: File::NULL) }
+    end
+    # rubocop:enable RSpec/BeforeAfterAll
```

Comment:
Related to my past comment, but maybe we can move this to a `shared_context` in `generator_examples.rb`?

Reply:
Fixed in fa9ffe528f